ios/FluentUI/Vnext/Avatar/AvatarGroup.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public enum MSFAvatarGroupStyle: Int, CaseIterable {
+    case stack
+    case pile
+}
+
+@objc public protocol MSFAvatarGroupState {
+    var maxDisplayedAvatars: UInt32 { get set }
+    var overflowCount: UInt { get set }
```

Comment:
The overflowCount should be calculated automatically based on the number of avatars that the group is holding.

<avatars_in_group> - maxDisplayedAvatars = overflow_count (if greater than zero)

Reply:
According to legacy:

> Calculation for the overflow count: max(avatars.count() - maxDisplayedAvatars, 0) + overflowCount.

Should it still be calculated here?

Reply:
Nevermind. I missed the point of overflowCount, which is to provide an addition to the avatars that were capped by the maxDisplayedAvatars.
I made another comment about the documentation of it. It's easy to get confused with this naming.
What are your thoughts on starting a discussion to rename this property to something like "additionalOverflowCount"?